vhost-user.yml

```diff
@@ -71,15 +71,13 @@
       name: "{{ ansible_facts['user_id'] }}"
       append: yes
     become: yes
-    register: libvirt_config
 
-# Restart libvirtd if something changed
-  - name: Restart libvirtd
+# libvirtd if something changed
+  - name: Start libvirtd
     service:
       name: libvirtd
-      state: restarted
+      state: started
```

Comment:
This won't restart the service if it was already running. So it won't pick up the configuration change

Reply:
There should be no need for restart now, am I wrong?

Reply:
There is if the user was added to the libvirt group in the previous task

Reply:
We should not need to restart service because of that:

```[root@virtlab ~]# useradd mrnobody
[root@virtlab ~]# su mrnobody
[mrnobody@virtlab root]$ env LIBVIRT_DEFAULT_URI=qemu:///system virsh list
==== AUTHENTICATING FOR org.libvirt.unix.manage ===
System policy prevents management of local virtualized systems
Authenticating as: root
Password: ^C
[mrnobody@virtlab root]$ exit

[root@virtlab ~]# usermod -aG 'libvirt' mrnobody
[root@virtlab ~]# su mrnobody
[mrnobody@virtlab root]$ env LIBVIRT_DEFAULT_URI=qemu:///system virsh list
 Id    Name                           State
----------------------------------------------------

[mrnobody@virtlab root]$ ```

But I can test the ansible script further if you found issues in the past.

Reply:
Fair enough :)